packages/components/ng-ovh-sidebar-menu/src/index.less

```diff
@@ -10,6 +10,7 @@
   bottom: 0;
   left: 0;
   padding: @sidebar-padding;
+  overflow: scroll;
```

Comment:
won't this display a scrollbar on desktop view ?

Reply:
Yep if the height go over container height.

Reply:
It displays scrollbar all the time. Better to use `overflow: auto` since it displays the scrollbar only when the content overflows.

Reply:
Yep, right, better with auto.
Thanks